### ISSUE TITLE
Suggestion: Support for GNU tar format

### ISSUE THREAD

**Issue by feichh** [kamranzafar/jtar#9]
Hello,
As mentioned in https://github.com/kamranzafar/jtar/issues/6, jtar is not fully compatible with the GNU tar format. As gnu tar finds use on popular linux distributions such as ubuntu, you may want to consider supporting it in the future. It is possible to create ustar format archives with the gnu tar tool, though it is not the default.

cheers

**Comment by shawn11ZX** [comment]
Yeah, I met the problem of having my file name truncated to at most 100 characters.
This article explains some history background:
http://ftp.gnu.org/old-gnu/Manuals/tar/html_mono/tar.html#SEC119
